api/response/response.go

```diff
@@ -27,6 +31,14 @@ func Write(w http.ResponseWriter, resp Response) {
 	return
 }
 
+func WriteErr(ctx context.Context, w http.ResponseWriter, resp Response) {
```

Comment:
I liked that the response package was pretty standalone and not tied into macaron, and wanted to keep it that way.